appkit/next/core/installation.mdx

```diff
@@ -353,3 +353,153 @@ const nextConfig = {
   referrerpolicy="strict-origin-when-cross-origin"
   allowfullscreen
 ></iframe>
+
+## Troubleshoot
+
+### In-app Browser Wallet Detection
+
+When integrating AppKit Web SDK within your mobile app's in-app browser, you need to handle deep links and Universal Links properly. By default, mobile operating systems restrict how these links work in in-app browsers.
+
+To enable wallet connections via deep links, you must explicitly configure your in-app browser to intercept and handle these links. The code examples below show how to properly handle deep links for popular wallet apps like MetaMask and Trust Wallet across different mobile platforms and frameworks.
+
```

Comment:
any way to not repeat this on every single page? 
also is this only for core packages? doesn't this affect regular ak as well?

Reply:
Previously we had common pages but for installation we don't have. But maybe instead I could create a separate Troubleshoot page and use it for every framework. Will check, thanks for suggestion 🙏🏽

Reply:
It's for Web SDK directly, shouldn't matter which Core or full version

Reply:
I guess there is no way to use the docs commonly as I've checked Mintlify docs, @rtomas do you know if it's possible?

Reply:
@enesozturk you can add a file in a shared directory and call it from all the SDK

```
import SmartSessions from "/snippets/appkit/shared/smart-sessions.mdx";

<SmartSessions />
```